strategy-network-or-cache/README.md

```diff
@@ -0,0 +1,17 @@
+# Network or cache
+The recipe provides a service worker trying to retrieve the most updated content
+but with no detriment of user experience.
+
+## Difficulty
+Beginner
+
+## Use Case
+You want to show the most updated content but not making the user to wait too
```

Comment:
"... the most up to date content, but you do not want to make the user wait for too long"